### ISSUE TITLE
Colons in SASS maps marked as error

### ISSUE THREAD

**Issue by marvinhagemeister** [Microsoft/vscode#7434]
- VSCode Version: 1.2.0 + 1.3.0-insiders
- OS Version: MacOS X 10.11.5

Steps to Reproduce:

1. Open/create scss file
2. Write a map, the first colon is marked as an error
3. Error message: `") expected"`

<img width="333" alt="screen shot 2016-06-08 at 21 43 33" src="https://cloud.githubusercontent.com/assets/1062408/15908611/e272477c-2dc2-11e6-8e3a-d76f23d7ff30.png">

Code from screenshot:
```scss
// Breakpoints
$breakpoints: (
    'xsmall':  21.25rem, // 340px
    'small':  40rem, // 640px
    'medium':  48rem, // 768px
    'large':  61.25rem, // 980px
    'xlarge':  82rem // 1312px
);
```

**Comment by glen-84** [comment]
Duplicate of #1758.

**Comment by marvinhagemeister** [comment]
Fixed in latest insider build 1.3.0-insiders 7d56f33e508930d3727064481a7525344a227f10

**Comment by aeschli** [comment]
@marvinhagemeister Sorry, no work has been done on adding SCSS map support recently.
